shared-ui/lib/data.ts

```diff
@@ -39,6 +39,7 @@ const mainSiteTabInfo: TabInfo[] = [
   { name: 'About', link: '/#about' },
   { name: 'Calendar', link: '/#calendar' },
   { name: 'FAQ', link: '/#faq' },
+  { name: 'Sponsor Us', link: '/sponsor-us' },
```

Comment:
Also just curious: is the sponsor us page going to be a separate new page? Or is it supposed to be linking to the existing sections on the home page?

Reply:
It is a separate page! That's why it doesn't have a # in the path